src/test/java/org/folio/rest/impl/EholdingsPackagesTest.java

```diff
@@ -477,6 +468,171 @@ public void shouldReturn400WhenPackagePostDataIsInvalid()throws URISyntaxExcepti
       .statusCode(HttpStatus.SC_BAD_REQUEST);
   }
 
+  @Test
+  public void shouldReturnDefaultResourcesOnGetWithResources() throws IOException, URISyntaxException {
```

Comment:
I feel like those tests only test conversion of mod-kb-ebsco url into RM API url,
since this conversion is only done in TitlesFilterableUrlBuilder, I think it's better to test those different cases directly in TitlesFilterableUrlBuilderTest, that way tests will be faster and will only break if url mapping breaks,
And then it's good to leave 1-2 wiremock tests to check that everything is wired correctly

Reply:
Ok, left 2 tests to confirm overall wiring to expected RM API response - moved other tests to TitlesFilterableUrlBuilderTest that were not redundant with any existing